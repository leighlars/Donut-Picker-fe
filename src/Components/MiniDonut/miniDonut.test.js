import React from 'react'
import {shallow} from 'enzyme'
import MiniDonut from './MiniDonut'


describe('MiniDonut', () => {
    let MockFn = jest.fn()
    let wrapper = shallow( <MiniDonut saveFavorites = {MockFn}/> )
    it('should matchsnapshot', () => {
        wrapper.instance().setState({fill: "Cyan"})
        expect(wrapper).toMatchSnapshot()
    });
    it('should call saveForites on click', () => {
        let button = wrapper.find(".mini-donut");
        button.simulate("click", null);
        expect(MockFn).toHaveBeenCalled();
    });
    
});