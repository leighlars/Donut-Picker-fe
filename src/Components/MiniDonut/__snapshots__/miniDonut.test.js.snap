// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MiniDonut should matchsnapshot 1`] = `ShallowWrapper {}`;

exports[`miniDonut should matchsnapshot 1`] = `ShallowWrapper {}`;
